dokang/cli.py

```diff
@@ -43,9 +44,12 @@ def index(settings, only_doc_set, force):
         api.index_document_set(index_path, info, force)
 
 
-def clear(settings, docset):
+def clear(settings, docset, purge=False):
     index_path = settings['dokang.index_path']
     api.clear_document_set(index_path, docset)
+    if purge:
+        logger.info('Purging document set "%s"', docset)
```

Comment:
Other loggers include a dot at the end of the sentence. ;)

Also, I am not sure if it's clear (for the user) what purging does. Perhaps we should log something more explicit like `Deleting files of document set "%s".` ?

Apart from that, LGTM.